### ISSUE TITLE
Advice: "Screen Locked", the default input focus can be set at the password form.

### ISSUE THREAD

**Issue by yaray** [kingToolbox/WindTerm#218]
When shift back to WindTerm window, and If the current panal is "Screen Locked", (Advice) the default input focus can be set at the password form.

**Comment by kingToolbox** [comment]
Yes, it should be so. I will fix this in the next prerelease version, thank you.
